File: code/43133.pas
program Bai133;
var n, c :integer;
begin
  readln(n);
  while n > 0 do 
    begin
      c := n mod 10;
      if c mod 2 = 0 then 
      begin
        writeln('NO');
        exit;
      end;
       n := n div 10;
    end;
  writeln ('YES');
end.

File: code/43133.cpp
#include <iostream>
using namespace std;

int main() 
{
    int n,c;
    cin>>n;
    while (n>0){
        c=n%10;
        if (c%2==0){
            cout<<"NO";
            return 0;
        }
        n=n/10;
    }
    cout<<"YES";
    return 0;
}